scripts/ffmpeg-transcoding-benchmark.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+
+source functions/all.sh
```

Comment:
That line don't work for me.
```suggestion
cd ../ffmpeg-segment-experiments
source functions/all.sh
cd ../scripts
```